business/workloads.go

```diff
@@ -391,7 +391,21 @@ func (in *WorkloadService) getParsedLogs(namespace, name string, opts *LogOption
 		if opts.IsProxy {
 			engardeParser := parser.New(parser.IstioProxyAccessLogsPattern)
```

Comment:
Not part of the PR, but just noticing we construct a new parser for every envoy log message, is that necessary or can we do it only once?